app/models/ip_address_matcher.rb

```diff
@@ -7,7 +7,8 @@ def initialize(terms)
   end
 
   def ===(other)
-    @cidrs.any? { |cidr| cidr.matches?(other) }
+    other = NetAddr::IPv4.parse(other)
```

Comment:
Prefer using a new variable (i.e. not re-using the variable)

Reply:
Yeah, I kind of agree - only this is simply re-casting one object as another class - the role the object plays is the same and so I think I'd use the same name anyway